sp/src/game/server/physics_prop_ragdoll.cpp

```diff
@@ -1557,6 +1557,16 @@ CBaseEntity *CreateServerRagdoll( CBaseAnimating *pAnimating, int forceBone, con
 	pRagdoll->CollisionProp()->SetCollisionBounds( mins, maxs );
 
 #ifdef MAPBASE
+	// If this was a NPC running a dynamic interaction, disable collisions with the interaction partner
```

Comment:
What about when the animation is complete? Would this make that NPC never collide with this ragdoll? Could that ever be an issue?

Reply:
The ragdoll will continue to have collisions disabled with the interaction partner, but I don't think it would be an issue. There is nothing which specifically relies on NPCs colliding with serverside ragdolls.